google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -62,7 +66,7 @@ class AggregationResult(object):
     :param value: The resulting read_time
     """
 
-    def __init__(self, alias: str, value: int, read_time=None):
+    def __init__(self, alias: str, value: int | float, read_time=None):
```

Comment:
You can just use `float` instead of `int | float` (float implies int)

Reply:
That's good to know! Replaced all occurrences.